xml/2018-10-14.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<email title="Weekly Digest">
+
+  <image
+    width="80"
+    src="https://i.imgur.com/5rJTZsq.jpg"
+    alt="Photo of digestive biscuits on a plate"
+    title=""/>
+  <hr/>
+
+  <text>
+    Hi again! It's time for another digest.
+
+    In the past week, we held four events: the Mums and Dads meeting, Linux talk,
+    Sponsors' Exhibition and the Bloomberg Decentralised App Development workshop.
+    These events were an overwhelming success with a total attendee count over all
+    four events of 1000. Thanks to everyone who participated and a special thanks
+    to all of our volunteers!
+
+    If you have any feedback about our events, feel free to [get in touch](mailto:docsoc@imperial.ac.uk).
+    Additionally, if you want to help out at our events, be sure to join our
+    [DoCSoc: Operations](https://www.facebook.com/groups/1509850436007222/) Facebook group.
+
+    In the coming week, we have our first inaugural bar night of the year, the
+    Welcome Bar Night, so stay tuned for more information and opportunities from
```

Comment:
the Welcome Bar Night, with more industrial events to follow, so stay tuned for...